<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>grid_geohash_form</class>
 <widget class="QDialog" name="grid_geohash_form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>165</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Geohash Grid Generator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>20</number>
     </property>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="4" column="0" colspan="2">
      <widget class="QProgressBar" name="status">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="LblStatus">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="LinOutputFolder">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="BtnOutputFolder">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="Label">
       <property name="text">
        <string>Select folder to save shapefile</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinPrecision">
       <property name="toolTip">
        <string>Geohash Precision [1,30]</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Precision</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="BtnApplyClose">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BtnApplyClose</sender>
   <signal>accepted()</signal>
   <receiver>grid_geohash_form</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnApplyClose</sender>
   <signal>rejected()</signal>
   <receiver>grid_geohash_form</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
